test/init_test.go

```diff
@@ -67,6 +67,16 @@ func setup(t *testing.T, fn ...func(*testing.T, *clients, string)) (*clients, st
 	return c, namespace
 }
 
+func setupStatic(t *testing.T) (*clients, string) {
+	t.Helper()
+	namespace := "faninfanout"
```

Comment:
I'm guessing this is for debugging only?

Reply:
upz, forgot to remove this, thanks!